### PR TITLE
Refactor/connector settings

### PR DESCRIPTION
**Pull request by Nullably**
**Before submitting this PR, please make sure**:

- [x] Your code builds clean without any errors or warnings
- [x] You are using approved title ("feat/", "fix/", "docs/", "refactor/")

**After creating this PR, please make sure:**

- [ ] You link the related GitHub issue or ticket

**A description of the changes proposed in the pull request**:
- Remove DEXES, they are actually ethereum wallet connectors
- Create TradeFeeType 
- For Ethereum wallet connectors, they are additional setting required in utils.py file

### CODE REVIEW

File: hummingbot/client/settings.py [hunk 1]
```diff
@@ -29,84 +33,75 @@
...
+class ConnectorSetting(NamedTuple):
+    name: str
+    type: ConnectorType
+    example_pair: str
+    centralised: bool
+    use_ethereum_wallet: bool
+    fee_type: TradeFeeType
+    fee_token: str
+    default_fees: List[Decimal]
+    config_keys: List[ConfigVar]
```
Comment (vic-en) [inline hummingbot/client/settings.py:51]: I think this should be `config_keys: Dict[str, ConfigVar]`

**Review by vic-en — APPROVED** [review]
LGTM

File: hummingbot/client/settings.py [hunk 2]
```diff
@@ -29,84 +33,75 @@
...
+            connector_settings[connector_dir.name] = ConnectorSetting(
+                name=connector_dir.name,
+                type=ConnectorType[type_dir.name.capitalize()],
+                centralised=getattr(util_module, "CENTRALIZED", True),
+                example_pair=getattr(util_module, "EXAMPLE_PAIR", ""),
+                use_ethereum_wallet=getattr(util_module, "USE_ETHEREUM_WALLET", False),
+                fee_type=fee_type,
+                fee_token=getattr(util_module, "FEE_TOKEN", ""),
+                default_fees=getattr(util_module, "DEFAULT_FEES", []),
+                config_keys=getattr(util_module, "KEYS", [])
```
Comment (vic-en) [inline hummingbot/client/settings.py:85]: Also, in addition to the initial comment `config_keys=getattr(util_module, "KEYS", {})`

File: hummingbot/client/settings.py (hunk 1 above)
Comment (Nullably) [reply hummingbot/client/settings.py:51]: Good catch, thanks.

**Review by vic-en — APPROVED** [review]
LGTM

**Pull request merged**